sql/src/SqlCAst.sk

```diff
@@ -451,6 +451,7 @@ class CAnd(CExpr<Int>, CExpr<Int>) extends CExpr<Int>
 class CUp<T: frozen>(CExpr<T>) extends CExpr<T>
 class CCoalesce<T: frozen>(Array<CExpr<T>>) extends CExpr<T>
 class CIIsNull(CExpr<Int>) extends CExpr<Int>
+class CILength(CExpr<String>) extends CExpr<Int>
```

Comment:
Should it be named `CSLength`?

Reply:
Ahh, good catch, I misinterpreted the convention here as that letter denoting the return type rather than the arg type.